source/_integrations/input_select.markdown

```diff
@@ -80,10 +80,26 @@ This integration provides three services to modify the state of the `input_selec
 | ------- | ---- | ----------- |
 | `select_option` | `option` | This can be used to select a specific option.
 | `set_options` | `options`<br>`entity_id(s)` | Set the options for specific `input_select` entities.
-| `select_previous` | | Select the previous option.
-| `select_next` | | Select the next option.
+| `select_first` | | Select the first option.
+| `select_last` | | Select the last option.
 | `reload` | | Reload `input_select` configuration |
 
+#### Service `input_select.next`
+
+Select the next option.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `cycle` | yes | Whether to cycle to the first value after the last. Default: `true`
+
+#### Service `input_select.previous`
```

Comment:
```suggestion
#### Service `input_select.select_previous`
```

Reply:
Oops! Too many `select`s ;-) Fixed and merged.